### DISCUSSION TITLE
question about the page of A-Larger-App

### DISCUSSION THREAD

**Discussion by wqhhust** [day8/re-frame#701] [Q&A]
In the following page
https://github.com/Day8/re-frame/wiki/A-Larger-App

It said to navigate

> Your app-db could have :active-panel key containing an id for the panel being displayed.

But all panel has it's own db.cljs, such as panel1/db.cljs,  panel2/db.cljs,... panelN/db.cljs.
Then where should I put app-db? Should I create a key value for :active-panel on each panelN/db.cljs?

**Comment by mike-thompson-day8** [comment]
@wqhhust  you can't ask these sorts of questions here.  This is for issues.  Please ask your questions in one of the help forums. We have had this discussion before.

**Comment by wqhhust** [comment]
sorry, I have thought the wiki could be more clear to understand and raise it as an issue. Ok I will ask on some forum.

**Comment by lhrkkk** [comment]
I want to ask this question too. And, Is there any forum for specially discussing re-frame?

**Comment by mike-thompson-day8** [comment]
@lhrkkk here's a list:  https://github.com/Day8/re-frame/blob/master/CONTRIBUTING.md#support-questions
